dspace-xmlui/src/main/webapp/WEB-INF/web.xml

```diff
@@ -101,19 +101,22 @@
   <!--
     - Declare a filter to force UTF-8 encoding for all servlet requests
     -->
-  <filter>
-    <filter-name>SetCharacterEncoding</filter-name>
-    <filter-class>org.dspace.app.xmlui.cocoon.SetCharacterEncodingFilter</filter-class>
```

Comment:
I would like to better understand why we are removing the custom `SetCharacterEncodingFilter` (which seems to have an "ignore" option to force encoding) in favor of Spring's `CharacterEncodingFilter`.

For historical purposes, `SetCharacterEncodingFilter` seems to have been created to resolve UTF-8 encoding issues on submission (https://jira.duraspace.org/browse/DS-132) and on the XMLUI homepage (https://jira.duraspace.org/browse/DS-112) which was related.

So, we need to carefully test that this change of filters doesn't result in encoding areas elsewhere in DSpace.